libs/account-lib/src/utils/custom.decorator.ts

```diff
@@ -0,0 +1,25 @@
+import { registerDecorator, ValidationArguments, ValidationOptions } from 'class-validator';
+
+export function IsHexPublicKey(validationOptions?: ValidationOptions) {
```

Comment:
might be nice to add some `validationOptions` in here, such as:

* `prefixRequired: boolean`
* `minByteLen: number`
* `maxByteLen: number`

Would make it easy to support different key algorithms in the future.

Reply:
I'll add this in the validation specific ticket.

As an unrelated note I think going forward we have to be more careful about adding new features and functionality in general. We have a lot of existing features that needs to be hardened and adding more would only make our life harder.

One question I usually like to think about is that how hard would it be for a future developer to change this to support that extra feature or data. If it is easy enough and the probability of this being changed in the near future is low enough. Usually keeping it simple and dumb might be preferable.